lib/methods_for_similarity.py

```diff
@@ -0,0 +1,8 @@
+def pokrycie_przedzialow(przedzial1, przedzial2):
+  # metoda przyjmuje jako argument dwie listy w postaci przedzialu liczbowego, tj. lista = [a, b], gdzie a <= b
+  # metoda zwraca liczbę z przedziału [0, 1]
+  min_l = min(przedzial1[0], przedzial2[0])
+  max_l = max(przedzial1[0], przedzial2[0])
+  min_p = min(przedzial1[1], przedzial2[1])
+  max_p = max(przedzial1[1], przedzial2[1])
+  return (min_p - max_l)/(max_p - min_l)
```

Comment:
Jest ryzyko dzielenia przez zero?

Reply:
Jeżeli listy zawierające te przedziały nie są puste, to raczej nie. Jeszcze sprawdzę i potwierdzę.

Reply:
Wprowadziłem dwie linijki które sprawiają, że jeżeli któryś z przedziałów $\[a, b\]$ nie spełnia warunku $a < b$, to metoda zwraca liczbę $0$.